node/zkv-deploy-weight-template.hbs

```diff
@@ -23,7 +23,11 @@ use core::marker::PhantomData;
 /// Weights for `{{pallet}}` using the zkVerify node and recommended hardware.
 pub struct ZKVWeight<T>(PhantomData<T>);
 
+{{#if (eq pallet "pallet_election_provider_support_benchmarking")}}
+impl<T: frame_system::Config> frame_election_provider_support::WeightInfo for ZKVWeight<T> {
+{{else}}
 impl<T: frame_system::Config> {{pallet}}::WeightInfo for ZKVWeight<T> {
+{{/if}}
```

Comment:
Maybe you can avoid this condition in template by change the name in in runtime lib.rs at `define_benchmarks` macro

Reply:
That's a great idea, done!